docs/content/latest/releases/releases-overview.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Supported and planned releases
+headerTitle: Supported and planned releases
+linkTitle: Supported and planned releases
+description: An overview of the latest and current stable releases, and planned releases.
+section: RELEASES
+menu:
+  latest:
+    identifier: releases-overview
+    weight: 2588 
+isTocNested: true
+showAsideToc: true
+---
+
+{{< note title="Important" >}}
+
+As of version `2.2`, Yugabyte is following a new release versioning convention for YugabyteDB releases.  For details, see [Release versioning](../versioning).
+
+{{< /note >}}
+
+## Supported stable and latest releases
+
+The following release series include:
+
+- [Stable](../versioning/#stable-releases): Supported for production deployments.
+- [Latest](../versioning/#latest-releases): Supported for development and testing only.
+
+[Release versioning](../versioning) gives details on the stable and latest release series numbering convention that started with `2.2`.
+
+[Planned releases](#planned-releases), in a section below, includes timelines for upcoming stable and latest release series.
+
+| Release series                    | Released     | Maintenance support ends | EOL          | Status                              |
```

Comment:
@stevebang It would be nice to have the status (Latest (development and testing), Current Stable (recommended for production), Stable in parenthesis next to the release series instead of a separate column.

Reply:
done